### PR TITLE
Solutions for issue #26

### PR DESCRIPTION
**Pull request by msotocalvo**
Here's a simple implementation of some customization features that allows the user to modified visual parameters of the CD. Regards

### CODE REVIEW

**Comment by msotocalvo** [comment]
Sorry, seems like I created the PR over an old fork I created before. Could you please check the updated PR? In this version I also included the possibility to customize the labels of the populations. Regards

File: autorank/_util.py [hunk 1]
```diff
@@ -502,59 +528,49 @@ def rankpos(rank):
...
     for l, r in groups:
-        plot_line([(rankpos(sorted_ranks[l]) - side, start),
-                   (rankpos(sorted_ranks[r]) + side, start)],
-                  linewidth=2.5)
+        plot_line([(rankpos(sorted_ranks[l]) - side, start), (rankpos(sorted_ranks[r]) + side, start)], linewidth=linewidth * 2)
         start += no_sig_height
 
-    return ax
+    # Add title to the diagram
+    ax.set_title(title, fontsize=fontsize + 15)
```
Comment (sherbold) [inline autorank/_util.py:571]: needs None check, so that no title is added if no title is passed as argument

File: autorank/_util.py [hunk 2]
```diff
@@ -323,30 +323,51 @@ def rank_bayesian(data, alpha, verbose, all_normal, order, rope, rope_mode, nsam
...
+                if i == 0:
+                    # comparison with "best"
+                    result_df.loc[result_df.index[j], 'p_equal'] = posterior_probabilities[1]
+                    result_df.loc[result_df.index[j], 'p_smaller'] = posterior_probabilities[0]
+                    result_df.loc[result_df.index[j], 'decision'] = _posterior_decision(posterior_probabilities, alpha)
+
+    # Create a DataFrame for posterior probabilities
+    posterior_probabilities_df = pd.DataFrame(posterior_probabilities_list)
+
+    # Export to Excel
```
Comment (sherbold) [inline autorank/_util.py:365]: this seems to be local code

File: autorank/_util.py [hunk 3]
```diff
@@ -323,30 +323,51 @@ def rank_bayesian(data, alpha, verbose, all_normal, order, rope, rope_mode, nsam
     sample_matrix = pd.DataFrame(index=reordered_data.columns, columns=reordered_data.columns)
     posterior_matrix = pd.DataFrame(index=reordered_data.columns, columns=reordered_data.columns)
     decision_matrix = pd.DataFrame(index=reordered_data.columns, columns=reordered_data.columns)
+    posterior_probabilities_list = []
+
     for i in range(len(data.columns)):
-        for j in range(i+1, len(reordered_data.columns)):
```
Comment (sherbold) [inline autorank/_util.py:327]: What are these changes? Are they only for the excel-export?

**Review by sherbold — CHANGES_REQUESTED** [review]
Looks in general good, but there are some aspects that break existing behavior, docstrings are missing, and I think some code from local modifactions for storing an excel file is also still in there

**Comment by sherbold** [comment]
@msotocalvo I am planning a release for mid/end of August to incorporate the latest small changes and fixed. I would love to include this is well, if the remaining issues are resolved.

**Comment by msotocalvo** [comment]
Dear Steffen,

Thank you for considering my suggestions. I have been quite busy but I will
work to solve your previous comments during next week.

Best regards

**Pull request closed without merging**